incubator/group/keeper.go

```diff
@@ -510,6 +519,13 @@ func (k Keeper) CreateProposal(ctx sdk.Context, accountAddress sdk.AccAddress, c
 	if err != nil {
 		return 0, errors.Wrap(err, "get group by account")
 	}
+
+	for i := range proposers {
+		if !k.groupMemberTable.Has(ctx, GroupMember{Group: g.Group, Member: proposers[i]}.NaturalKey()) {
+			return 0, errors.Wrapf(ErrUnauthorized, "not in group: %s", proposers[i])
```

Comment:
can we avoid returning ProposalID as `0` on error. How about named return values?

Reply:
I try to follow the [Go code review](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters) comments and avoid named return values. If there is some other standard within regen codebase, please let me know.